server/accounts.go

```diff
@@ -758,6 +764,11 @@ type remoteLatency struct {
 // sendLatencyResult will send a latency result and clear the si of the requestor(rc).
 func (a *Account) sendLatencyResult(si *serviceImport, sl *ServiceLatency) {
 	si.acc.mu.Lock()
+
+	sl.Type = ServiceLatencyType
+	sl.ID = a.srv.nextEventID()
+	sl.Time = time.Now().UTC().Format(time.RFC3339Nano)
```

Comment:
Don't need the format and can just be time.Time, its json marshaller will use RFC3339Nano by default, so just need UTC() on normal time.Time.

Reply:
I know it does the right thing for time.Time but I like that when I am reading the code, I know what is happening. No question here what is in the time field and what format.

Someone reading the code who have not gone to read the code for time.MarshalJSON() will not know this and wonder - performance wise its about the same cos they would do this too and same allocations

Plus this is consistent with the rest of the code.

Reply:
ok but if the server ever wants to use this internally it would have to parse it back into a time.Time which is why I am pushing to keep things time.Time in code.

Reply:
OK, I'll update this PR to update all the other ones then too

Reply:
I think that is best. I understand where you are coming from but being in the server and going back and forth is painful.